CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## UNRELEASED
 
 ### Added
+
+- Add a run operator `A.expect(P)` to evaluate the result of `A` with `P` (#1303)
```

Comment:
Something like this would be clearer IMO:


```suggestion
- Add a run operator `A.expect(P)` to test the state predicate `P` in the state resulting from applying action `A` (#1303)
```